float64 serial_msg_hz
uint8 mode
uint8 e_stop
uint8 gear
uint16 speed
int16 steer
uint8 brake
uint8 alive
